.gitignore

```diff
@@ -13,3 +13,4 @@ ios/send-ios/assets/vendor.js
 ios/send-ios.xcodeproj/project.xcworkspace/xcuserdata/*
 ios/send-ios.xcodeproj/xcuserdata/*
 test/integration/downloads
+package-lock.json
```

Comment:
`package-lock` should generally be committed

Reply:
should't the deployment pipeline be responsible of installing & locking dependencies as opposed to devs?

Reply:
committing it should ensure that what we deploy has the same dependencies that we tested with/developed on. Maybe a case for a larger discussion?

Reply:
removing package-lock.json to allow npm ci to build from package-lock.json (as opposed to building from package.json)